packages/clients/meldemichel/src/mapConfigurations.ts

```diff
@@ -123,8 +123,11 @@ const geoLocation: Partial<GeoLocationConfiguration> = {
   showTooltip: true,
 }
 
-const mapConfigurations = {
-  [MODE.COMPLETE]: (reportServiceId: string) => {
+const mapConfigurations: Record<
+  string,
```

Comment:
```suggestion
  keyof typeof MODE,,
```
Also, `[MODE.COMPLETE]` can be simpliefied to `COMPLETE`

Reply:
>Also, `[MODE.COMPLETE]` can be simpliefied to `COMPLETE`

It can be, but I figured relating to the ENUM indicates more obviously what's going on.

Reply:
By changing the type to `keyof typeof MODE` that is still the case, even if the enum is not used for all the modes.

Though, if you insist on keeping it that way, I won't press you 🎩